src/content/docs/usage/ide.mdx

```diff
@@ -61,11 +54,299 @@ You can set the default IDE used by Daytona.
+
+   ```shell
+   daytona ide
+   ```
+
+    Upon running this command, Daytona will display a list of available IDEs, allowing you to select `PyCharm Professional` as your default choice.
+
+    ```text
+      Choose Your Default IDE
+
+      | PyCharm Professional
+    ```
+
+2. Select `PyCharm Professional` from the list presented. Upon selecting, Daytona will set it as the default IDE and display a confirmation message.
+
+    ```text
+    Default IDE: PyCharm Professional
+    ```
+
+    You can now use the `daytona code` command to automatically open the JetBrains PyCharm Professional IDE already connected to your Workspace.
+
+## JetBrains Rider
+
+Daytona allows you to connect to your Workspace using JetBrains Rider IDE.
+
+1. Run the following command to set your default IDE to JetBrains Rider:
+
+   ```shell
+   daytona ide
+   ```
+
+    Upon running this command, Daytona will display a list of available IDEs, allowing you to select `Rider` as your default choice.
+
+    ```text
+      Choose Your Default IDE
+
+      | Rider
+    ```
+
+2. Select `Rider` from the list presented. Upon selecting, Daytona will set it as the default IDE and display a confirmation message.
+
+    ```text
+    Default IDE: Rider
+    ```
+
+    You can now use the `daytona code` command to automatically open the JetBrains Rider IDE already connected to your Workspace.
+
+## JetBrains RubyMine
+
+Daytona allows you to connect to your Workspace using JetBrains RubyMine IDE.
+
+1. Run the following command to set your default IDE to JetBrains RubyMine:
+
+   ```shell
+   daytona ide
+   ```
+
+    Upon running this command, Daytona will display a list of available IDEs, allowing you to select `RubyMine` as your default choice.
+
+    ```text
+      Choose Your Default IDE
+
+      | RubyMine
+    ```
+
+2. Select `RubyMine` from the list presented. Upon selecting, Daytona will set it as the default IDE and display a confirmation message.
+
+    ```text
+    Default IDE: RubyMine
+    ```
+
+    You can now use the `daytona code` command to automatically open the JetBrains RubyMine IDE already connected to your Workspace.
+
+## JetBrains WebStorm
+
+Daytona allows you to connect to your Workspace using JetBrains WebStorm IDE.
+
+1. Run the following command to set your default IDE to JetBrains WebStorm:
+
+   ```shell
+   daytona ide
+   ```
+
+    Upon running this command, Daytona will display a list of available IDEs, allowing you to select `WebStorm` as your default choice.
+
+    ```text
+      Choose Your Default IDE
+
+      | WebStorm
+    ```
+
+2. Select `WebStorm` from the list presented. Upon selecting, Daytona will set it as the default IDE and display a confirmation message.
+
+    ```text
+    Default IDE: WebStorm
+    ```
+
+    You can now use the `daytona code` command to automatically open the JetBrains WebStorm IDE already connected to your Workspace.
+
+## Terminal SSH
+
+Daytona allows you to connect to your Workspace using Terminal SSH.
```

Comment:
`Daytona allows you to connect to your Workspace using Terminal SSH.` => `Daytona allows you to connect to your Workspace through SSH`.